Formula/swagger-codegen.rb

```diff
@@ -18,8 +18,7 @@ class SwaggerCodegen < Formula
 
   def install
     # Need to set JAVA_HOME manually since maven overrides 1.8 with 1.7+
-    cmd = Language::Java.java_home_cmd("1.8")
-    ENV["JAVA_HOME"] = Utils.safe_popen_read(cmd).chomp
+    ENV["JAVA_HOME"] = Language::Java.java_home("1.8")
```

Comment:
The github page mentions that it needs to be at least Java 8, so we're better of making the formula depend on `openjdk` and using `ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix` here.

Reply:
thanks @SMillerDev 
I guess that would give the install location of openjdk - given that it's installed?
Is it a good practice to depend on other formulae? The formula depends on java 1.8 - is it enough to say `openjdk`? Shouldn't we restrict the version? Will `java` always be the same as `openjdk`? And why the hell do the tests fail? :|

Reply:
- I guess that would give the install location of openjdk - given that it's installed?
That's what `ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix` will do

- Is it a good practice to depend on other formulae?
It's an excellent practice and the basis of homebrew. There's very few formula that don't do this.

- The formula depends on java 1.8 - is it enough to say openjdk?
You can replace the line `depends_on java` with `depends_on "openjdk"`

- Shouldn't we restrict the version?
No, homebrew always strives to use the latest dependencies. Restricting versions would make that more difficult.

- Will java always be the same as openjdk?
Yes, OpenJDK is an implementation of Java (and one that homebrew ships as a formula)

- And why the hell do the tests fail? :|
Gonna restart them to try again

Reply:
Actually, if you push those changes it should restart again.

Reply:
Aight, let's see if the tests run fine now.